vendor/bat-native-ads/src/bat/ads/internal/supported_regions.h

```diff
@@ -0,0 +1,325 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_ADS_INTERNAL_SUPPORTED_REGIONS_H_
+#define BAT_ADS_INTERNAL_SUPPORTED_REGIONS_H_
+
+#include <map>
+#include <set>
+#include <string>
+
+namespace ads {
+
+const std::map<int, std::set<std::string>> kSupportedRegions = {
```

Comment:
Global `std::map` is prohibited by https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables.
Read "Common patterns" section for a solution: array of pairs.
constexpr std::array<std::pair<int, const std::array<const char*, M>, N>.
Keep the collection in sorted order and use a binary search algorithm.

Reply:
I raised https://github.com/brave/brave-browser/issues/9164 to fix this globally in Brave Ads